ui/app/pages/first-time-flow/create-password/new-account/new-account.component.js

```diff
@@ -211,7 +211,7 @@ export default class NewAccount extends PureComponent {
                 rel="noopener noreferrer"
               >
                 <span className="first-time-flow__link-text">
-                  { 'Terms of Use' }
+                  Terms of Use
```

Comment:
should we be using translations for these?

Reply:
We should, yes

Reply:
Fixed